src/clients/deep-infra.ts

```diff
@@ -1,16 +1,21 @@
 import axios, {AxiosInstance, AxiosRequestConfig, AxiosResponse} from 'axios';
 import {INITIAL_BACKOFF, MAX_RETRIES, SUBSEQUENT_BACKOFF, USER_AGENT} from '@/lib/constants/client';
+import {ClientConfig} from "@/lib/types/common/client-config";
 
 export class DeepInfraClient {
   private axiosClient: AxiosInstance;
-  private readonly maxRetries: number = MAX_RETRIES;
-  private readonly initialBackoff: number = INITIAL_BACKOFF;
-  private readonly subsequentBackoff: number = SUBSEQUENT_BACKOFF;
+  private readonly maxRetries: number
```

Comment:
I'd just stick a (non-optional) config object here, so if we add stuff later you won't have to change anything.